lddtree.sh

```diff
@@ -69,9 +68,11 @@ elf_specs_scanelf() {
 		sed -r 's: (LINUX|GNU)$: NONE:'
 }
 
+
+
 # functions for binutils backend
```

Comment:
to separate a bit more binutils/readelf, and scanelf backend
for more readability, in my point of view...